docs/huggingface/tgi-0.9.3.md

```diff
@@ -0,0 +1,42 @@
+| NGROK_AUTHTOKEN          |                             | ngrok authentication token                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| NGROK_DOMAIN             |                             | ngrok domain name where the axum webserver will be available at                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| NGROK_USERNAME           |                             | ngrok basic auth username                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
missing last row which is `NGROK_PASSWORD`. Description: `ngrok basic auth password`